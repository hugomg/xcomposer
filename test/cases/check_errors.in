# Large character
U0041 AB aa

# Wrong codepoint
U0041 z z

# Incompatibility
U0041 A a_ 
U0041 A a_
