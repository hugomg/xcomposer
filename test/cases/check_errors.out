xcomposer encountered an error:

Line 2: Rule does not output a single utf8 codepoint
Line 5: Wrong character. Expected 'A' (U0041), got 'z' (U007A)
Incompatible rules:
  Line 8: a_
  Line 9: a_
